app/code/Magento/Ui/view/base/web/js/core/renderer/layout.js

```diff
@@ -600,6 +602,25 @@ define([
             });
 
             return result;
+        },
+
+        /**
+         * Get sort order for child if it is undefined.
+         *
+         * @param {Object} parent
+         * @param {Object} child
+         */
+        getChildSortOrderIfUndefined: function (parent, child) {
+            var key, i = 0;
```

Comment:
While `i` is a common name for index in the loops, it is still suggested to use conscious names.

Reply:
Fixed.